third_party/openthread/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ set(OT_BUILD_EXECUTABLES OFF CACHE BOOL "disable building executables" FORCE)
 set(OT_BUILTIN_MBEDTLS_MANAGEMENT OFF CACHE BOOL "diable mbedTLS management" FORCE)
 set(OT_COMMISSIONER ON CACHE BOOL "enable commissioner" FORCE)
 set(OT_DAEMON ON CACHE BOOL "enable daemon mode" FORCE)
-set(OT_FIREWALL ON CACHE BOOL "enable firewall feature" FORCE)
+set(OT_FIREWALL ON CACHE BOOL "enable firewall feature")
```

Comment:
@wgtdkp had this suggestion: https://github.com/openthread/ot-br-posix/pull/1195#discussion_r783626233

Reply:
We had an offline discussion and agreed not to add a comment since no such comment has been added on other options.